src/ig_subscriptions/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (context, req) {
```

Comment:
@nastassiar need to hook this up to the site so that when a user signs up we create an instagram subscriptioin